src/components/mdx/image-zoom.tsx

```
@@ -0,0 +1,92 @@
+  width,
+  height,
+  caption,
+}: ImageZoomProps) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      {/* Thumbnail */}
+      <figure className="my-8">
+        <div
+          role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)